modules/apps/oauth2-provider/oauth2-provider-service/service.xml

```diff
@@ -183,5 +183,6 @@
 		<exception>OAuth2ApplicationRedirectURIMissing</exception>
 		<exception>OAuth2ApplicationRedirectURIPath</exception>
 		<exception>OAuth2ApplicationRedirectURIScheme</exception>
+		<exception>OAuth2ApplicationRequired</exception>
```

Comment:
Add it as the first commit and the next commit should be a "gw buildService" one.